package.json

```diff
@@ -43,7 +43,7 @@
     "ember-blog": "link:./tests/dummy/lib/ember-blog",
     "ember-chat": "link:./tests/dummy/lib/ember-chat",
     "ember-cli": "~3.7.1",
-    "ember-cli-addon-tests": "^0.11.0",
+    "ember-cli-addon-tests": "github:kiwiupover/ember-cli-addon-tests#test-against-ember-data-release",
```

Comment:
What is the meaning of this PR? What does it solve?

we should use stable releases here

Reply:
closed by #639